draft-ietf-quic-multipath.md

```diff
@@ -382,55 +382,50 @@ Reset ({{Section 10.3 of QUIC-TRANSPORT}}) closes the connection.
 
 Both endpoints, namely the client and the server, can close a path,
 by sending PATH_ABANDON frame (see {{path-abandon-frame}}) which
-abandons the path with a corresponding Path Identifier. Once a path is
-marked as "abandoned", it means that the resources related to the path,
-such as the used connection IDs, can be released.
-However, information related to data delivered over that path SHOULD
-not be released immediately as acknowledgments can still be received
-or other frames that also may trigger retransmission of data on another
-path.
+abandons the path with a corresponding Path Identifier.
 
-The endpoint sending the PATH_ABANDON frame SHOULD consider a path as
+The endpoint sending the PATH_ABANDON frame can consider a path as
 abandoned when the packet that contained the PATH_ABANDON frame is
-acknowledged. When releasing resources of a path, the endpoint SHOULD
-send a RETIRE_CONNECTION_ID frame for the connection IDs used on the path,
-if any.
-
-The receiver of a PATH_ABANDON frame SHOULD NOT release its resources
-immediately, but SHOULD wait for the reception of the RETIRE_CONNECTION_ID
-frame for the used connection IDs or 3 RTOs.
+acknowledged.
+
+The receiver of a PATH_ABANDON frame should not release its resources
+immediately, but SHOULD wait for at least three times the current
+Probe Timeout (PTO) interval as defined in {{Section 6.2. of QUIC-RECOVERY}}
+after the last sent packet before sending the RETIRE_CONNECTION_ID frame
+for the corresponding CID.
+This is inline with the requirement of {{Section 10.2 of QUIC-TRANSPORT}}
+to ensure that paths close cleanly and that delayed or reordered packets
+are properly discarded.
+The effect of receiving a RETIRE_CONNECTION_ID frame is specified in the
+next section.
```

Comment:
This is a new normative requirement now but I think this is what is actually meant by "releasing resources". Particularly you should only retire you CID if you are sure you don't need it anymore. Hence the new normative requirement is to wait 3 PTOs before you sent the retired CID frame after receiving the abandon frame.

Reply:
Yes, this is good. There are two big motivations: packets may be delivered out of order, with packets in transit arriving after the PATH_ABANDON; and, acknowledgements still need to be sent. Waiting 3 PTO is the standard mitigation for such issues.

Reply:
The suggestion above should address the point raised by @Yanmei-Liu in the resolved conversation below.